StackExRedis/NHibernate.Caches.StackExRedis/NHibernate.Caches.StackExRedis.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Iesi.Collections" Version="4.0.4" />
     <PackageReference Include="NHibernate" Version="5.2.0" />
-    <PackageReference Include="StackExchange.Redis.StrongName" Version="1.2.6" />
+    <PackageReference Include="StackExchange.Redis" Version="2.0.495" />
```

Comment:
With version 2.0 the `.StrongName` package does not exist anymore as `StackExchange.Redis` is now strong-named.